### ISSUE TITLE
[Bug] Import of prettier plugin

### ISSUE THREAD

**Issue by safraja** [ng-doc/ng-doc#136]
### Is this a regression?

I don't know

### Description

Hi,
I'm trying to use NgDoc for our new project (under Nx), but I'm having some trouble. We are using prettierr and in it we have

> "plugins":["prettier-plugin-brace-style"],

And this line is causing an error that I don't know how to deal with. Could someone please help me?

### Please provide the exception or error you saw

```true
NgDoc: Post-build...file:///...node_modules/prettier/index.mjs:22053
      throw new UndefinedParserError(
            ^
   - documentation:serve:development
UndefinedParserError: No parser and no file path given, couldn't infer a parser.
    at normalizeFormatOptions (file:///...)
    at formatWithCursor (file:///...)
    at file:///...
    at async Module.format2 (file:///...)
    at async Object.parse (...)

Node.js v20.9.0lizing...
```

### OS

Windows

### Browser

None

### Node version

20.9

**Comment by safraja** [comment]
Sure, it is 3.2.5. I suppose the others won't help you, but here's a list of everything prettier related from our package.json:

> "prettier": "^3.2.5",
> "prettier-plugin-brace-style": "^0.5.0",
> "prettier-eslint": "^16.1.2",
> "eslint-config-prettier": "^9.1.0",
> "eslint-plugin-prettier": "^5.0.1",

**Comment by skoropadas** [comment]
Thanks, it's probably the version of prettier, ngdoc uses 2.x.x, and I think that's the problem, I'll fix it tomorrow, thanks again!

**Comment by skoropadas** [comment]
should be fixed now, pls let me know if you still have this problem

**Comment by safraja** [comment]
Thank you very much, it looks like it's working!

Also thanks for the explanation, I was wondering what does the ngdoc need prettier for. When I wasn't able to do anything about the error, I was wondering if it was possible to force ngdoc to ignore prettier or use another config file where I could just omit the problematic plugin. But I couldn't manage to do either, so I finally asked for help here.

As for the versions, as I wrote earlier, we're working on a brand new project, so we're using the latest versions of basically everything. Because of this, I also ran into some complications with ngdoc regarding InputSignals. I'll probably create a new ticket for that. 🙂
